benchmark_test.go

```diff
@@ -218,3 +220,76 @@ func TestBenchmarkStatsPerPeer(t *testing.T) {
 	})
 	assert.Equal(t, want, statsServer.Aggregated(), "unexpected stats")
 }
+
+// Method that tests the 'Format' option of Benchmark Options
+func TestBenchmarkOutput(t *testing.T) {
+	// Testing both plaintext and JSON output
+	tests := []struct {
+		opts BenchmarkOptions
```

Comment:
a couple of things for writing a good table test:
- we can add a field to indicate the test case name, and use `t.Run` so that each test case can be run independently, and easily readable from the test output, an good example: https://github.com/yarpc/yab/blob/dev/template_test.go#L159
- we can add a filed in the test struct to indicate what are the expected output, and can just use that in the test, that way, you can get ride of the if-else check